src/registrar/views/domain.py

```diff
@@ -919,11 +925,9 @@ def form_valid(self, form):
                 role=UserDomainRole.Roles.MANAGER,
             )
         except IntegrityError:
-            # User already has the desired role! Do nothing??
-            pass
-
-        messages.success(self.request, f"Added user {requested_email}.")
-
+            messages.warning(self.request, f"{requested_email} already added to this domain")
```

Comment:
@asaki222 This scenario seems similar to user already accepting a domain invitation (line 848 in the code). i.e. they're already a domain manager. Is that correct? 

If so, then I think the alert message should match that other scenario: 

> "{email} is already a manager for this domain."

Reply:
@Katherine-Osos updated the text!